eng/pipelines/templates/Build_and_UnitTest.yml

```diff
@@ -309,6 +309,13 @@ steps:
     TargetFolder: "$(BuildOutputTargetPath)\\artifacts\\$(VsixPublishDir)\\$(NupkgOutputDir)"
   condition: and(succeeded(), or(eq(variables['IsOfficialBuild'], 'true'), eq(variables['BuildRTM'], 'true')))  #skip this task for nonRTM in private build
 
+- task: PublishPipelineArtifact@1
+  displayName: "Publish nupkgs"
+  inputs: 
+    targetPath: "$(Build.Repository.LocalPath)\\artifacts\\$(NupkgOutputDir)"
```

Comment:
There's also a step `displayName: "Copy NuGet.exe, VSIX and EndToEnd"`
which copies from the VSIX, nuget.exe, etc from a higher folder. Will those be captured as artifacts somewhere?

Reply:
> publish artifacts\vs15 as pipeline artifact on both rtm and non-rtm builds

where's this?

Reply:
Lines 427 to 433 define the "publish artifacts\VS15 directory". I deleted the condition that only does it on the non-RTM build, so we get RTM artifacts as well.

Reply:
Line 427 shows as `'OptProfV2:  publish profiling inputs to artifact services'` for me, atm.
Anyway, I do see the VSIX/exe in your run's artifacts.
I do not see published nupkgs, though 😮

Reply:
Sorry, like 427-433 in the "previous" version, it's likes 434-439 in the new file.

Reply:
On private builds we don't generate nupkgs in the non-rtm build (finish as quickly as possible, so mac, apex, and e2e tests can start)